tests/converters/herrenberg_bike_test.py

```diff
@@ -0,0 +1,59 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from pathlib import Path
+from unittest.mock import Mock
+
+import pytest
+from parkapi_sources.converters import HerrenbergBikePullConverter
+from requests_mock import Mocker
+
+from tests.converters.helper import validate_realtime_parking_site_inputs, validate_static_parking_site_inputs
+
+
+@pytest.fixture
+def requests_mock_herrenberg_bike(requests_mock: Mocker) -> Mocker:
+    json_path = Path(Path(__file__).parent, 'data', 'herrenberg_bike.json')
+    with json_path.open() as json_file:
+        json_data = json_file.read()
+
+    requests_mock.get('https://www.munigrid.de/api/file/11/latest', text=json_data)
+
+    return requests_mock
+
+
+@pytest.fixture
+def herrenberg_bike_pull_converter(mocked_config_helper: Mock) -> HerrenbergBikePullConverter:
+    return HerrenbergBikePullConverter(config_helper=mocked_config_helper)
+
+
+@pytest.fixture
+def herrenberg_bike_ignore_missing_capacity_config_helper(mocked_config_helper: Mock):
```

Comment:
Please use this new config helper in a test, so that you can see that the logic works.

Reply:
Now tested and seems to work.